x-pack/plugins/uptime/server/lib/adapters/monitors/elasticsearch_monitors_adapter.ts

```diff
@@ -160,13 +160,6 @@ export class ElasticsearchMonitorsAdapter implements UMMonitorsAdapter {
                     },
                   },
                 },
-                {
```

Comment:
I don't think this was correct before. A monitor's URL can change, but it shouldn't affect the count.